### ISSUE TITLE
govc: create a datastore cluster

### ISSUE THREAD

**Issue by marema31** [vmware/govmomi#1227]
Hello,

Is there a way to create a datastore cluster with govc ?

**Comment by dougm** [comment]
Yes: https://github.com/vmware/govmomi/blob/master/govc/USAGE.md#foldercreate
After creating, use `object.mv` to put datastores into the cluster: https://github.com/vmware/govmomi/blob/master/govc/USAGE.md#objectmv
